modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+Management cluster:: An {product-title} cluster where the HyperShift Operator is deployed and where the control planes for tenant clusters are hosted.
+
+Hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+
+Hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+Hosted control plane infrastructure:: Resources on the management cluster or external cloud provider that are prerequisites to run hosted control plane processes.
+
+Management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
+
+Hosted cluster infrastructure:: Network, compute, and storage resources that exist in the tenant or end-user cloud account.
+
+Cluster service provider:: This role is assumed by a user who hosts the cluster control planes. Typically, users who assume this role are cluster administrators on the management cluster and have Role-Based Access Control to monitor and own the availability of the HyperShift Operator and the control planes for the tenant's hosted clusters. The cluster service provider persona is responsible for the following activities:
+** Maintaining uptime by observing the user interface for fleet-wide alerts
```

Comment:
```suggestion
** Maintaining uptime by observing the user interface for fleet-wide alerts
```

Which user interface is not that clear.

 We could say, owning service-level objects for control-planes availability, uptime, stability, ...